trackintel/io/postgis.py

```diff
@@ -92,7 +92,7 @@ def read_positionfixes_postgis(
         If specified, return an iterator where chunksize is the number
         of rows to include in each chunk.
 
-    **kwargs
+    read_gpd_kws : dict, default None
         Further keyword arguments as available in trackintels trackintel.io.read_positionfixes_gpd().
         Especially useful to rename column names from the SQL table to trackintel conform column names.
         See second example how to use it in code.
```

Comment:
I'm a bit confused by the name `read_gpd_kws` - can you put any other kwargs instead of renaming? Because if it's only used for renaming columns, then I would call it `rename_columns` or `rename_dict` or so. If we call it `read_gpd_kws`, it sounds like this dictionary is passed on to the `geopandas.from_postgis` function, but it's not, right?

Reply:
Yeah, I had some problems naming it. It can be used for all the keyword arguments of the `read_xyz_gpd` functions.
Which is mostly used to rename columns, it can be used for other things but that wouldn't make sense with the interaction between the postgis and the gpd function. But if you want to rename other columns then one has to past an extra mapper in.

Reply:
I see, I guess we can't avoid that users have to look at the read_xzy_gpd functions in order to set these kw arguments. Fine by me